ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -429,6 +429,7 @@ describe('RuntimeConfigurationPanel', () => {
       status: RuntimeStatus.Deleted,
       configurationType: RuntimeConfigurationType.HailGenomicAnalysis,
       gceConfig: null,
+      gceWithPdConfig: null,
```

Comment:
test cleanup - only one of [gceConfig,gceWithPdConfig,dataprocConfig] should be defined.  These tests clearly intend that a different one should be defined.

Reply:
Nice.